api/cloud.proxy.js

```diff
@@ -43,6 +47,7 @@ function IoTKitCloud(logger, deviceId, customProxy) {
     me.deviceId = deviceId;
     me.deviceName = deviceConf.device_name;
     me.gatewayId = deviceConf.gateway_id || deviceId;
+    me.store = Sensor.init("device.json", me.logger);
```

Comment:
just a question: does this init reinitialize device.json? Also - use already defined filename instead of magic string